src/ui/assets/AssetsListContainer.js

```diff
@@ -84,7 +84,7 @@ export const mapDispatchToProps = (dispatch, ownProps) => ({
       dispatch(fetchAssetsPaged(undefined, undefined, ownerGroup, joinGroups));
     }
   },
-  onSetColumnOrder: columnOrder => dispatch(setColumnOrder(columnOrder, 'assetsList')),
+  onSetColumnOrder: columnOrder => ownProps.showColumns && dispatch(setColumnOrder(columnOrder, 'assetsList')),
```

Comment:
would it work for regular AssetsList page?
since it won't have ownProps.showColumns exist on that page the column order setting will work? @jeffgo10

Reply:
ah you're right @ichalagashvili , this is supposed to be negated. Thanks for spotting this